Gemfile

```diff
@@ -1,12 +1,13 @@
-ruby "2.2.4"
+ruby RUBY_VERSION
```

Comment:
I've not seen people do this before - the `ruby` declaration tends to be used as a declaration of the version of Ruby that the project is intended for. I see it as a useful signal of that.

The [Bundler documentation](http://bundler.io/v1.3/gemfile_ruby.html) seems to suggest that this is useful if you depend on features of specific Ruby implementations though (which I'm pressure sure we don't, so maybe this is fine).